style.css

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 400px;
+  margin: auto;
+  height: 4910px;
+  position: relative;
+  background: #fff;
+}
+
+.geomtry2 {
+  position: absolute;
+  width: 100%;
+  height: 160px;
+  left: 0;
+  top: 472px;
+}
+
+.menu {
+  top: 19px;
+  right: 10px;
+  width: 20px;
+  height: 14px;
+  position: absolute;
+}
+
+.title {
+  position: absolute;
+  width: 220px;
+  height: 100px;
+  left: 16px;
+  top: 104px;
+  font-family: 'Playfair Display', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
```

Comment:
[OPTIONAL] Again please :pray: do not use the exact styling properties from figma they are not responsive or correct, rather try to style elements using your flex or other css skills, It will help you a lot in the long run :+1: